packages/workbox-background-sync/test/browser/background-sync-queue.js

```diff
@@ -71,17 +74,29 @@ describe('background sync queue test', () => {
 
   it('check push proxy', async () => {
     const currentLen = backgroundSyncQueue._queue.queue.length;
-    await backgroundSyncQueue.pushIntoQueue({request: new Request('http://lipsum.com')});
+    await backgroundSyncQueue.pushIntoQueue({request: new Request('http://localhost:3000/__echo/counter')});
+    await backgroundSyncQueue.replayRequests();
```

Comment:
cleaned this up, made `bgSyncQueue` to be reinitiated before every test, so that replay is not needed anymore.